test/test_optim.py

```diff
@@ -609,6 +691,31 @@ def test_adadelta(self):
             with self.assertRaisesRegex(ValueError, "Invalid rho value: 1.1"):
                 optimizer(None, lr=1e-2, rho=1.1)
 
+    # new test that test_adadelta can be switched to when merge is complete and multitensor is deleted
+    @skipIfRocm
+    def test_adadelta_new(self):
```

Comment:
What is this test covering that is not covered above?

Reply:
It is the same test, the above one tests optim.Adadelta and optim_mt.Adadelta, this one tests foreach=True and False instead (if we go ahead with deleting the _multitensor directory, I was intending to add a pr to the top of the stack that copies the logic from `test_*_new` into `test_*`, does that make sense?